go/tasks/plugins/k8s/kfoperators/common/common_operator.go

```diff
@@ -0,0 +1,125 @@
+package common
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	commonOp "github.com/kubeflow/tf-operator/pkg/apis/common/v1"
+	logUtils "github.com/lyft/flyteidl/clients/go/coreutils/logs"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	flyteerr "github.com/lyft/flyteplugins/go/tasks/errors"
+	"github.com/lyft/flyteplugins/go/tasks/logs"
+	pluginsCore "github.com/lyft/flyteplugins/go/tasks/pluginmachinery/core"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	TensorflowTaskType = "tensorflow"
+	PytorchTaskType    = "pytorch"
+)
+
+func ExtractCurrentCondition(jobConditions []commonOp.JobCondition) (commonOp.JobCondition, error) {
+	sort.Slice(jobConditions[:], func(i, j int) bool {
```

Comment:
Does this serve any function?
```suggestion
	sort.Slice(jobConditions, func(i, j int) bool {
```

Reply:
Done